nengo/tests/test_params.py

```diff
@@ -275,3 +277,25 @@ class Test(object):
     # Not OK: not a function
     with pytest.raises(ValidationError):
         inst.fp = 0
+
+
+def test_params():
```

Comment:
Maybe call this `test_obsolete_excluded` or something? `test_params` seems to general.